device-types/Cisco/WS-C3850-12XS-S.yaml

```diff
@@ -0,0 +1,45 @@
+---
+manufacturer: Cisco
+model: Catalyst 3850-12XS-S
+slug: cisco-ws-c3850-12xs-s
+part_number: WS-C3850-12XS-S
+is_full_depth: false
+u_height: 1
+comments: IP Base feature set
+interfaces:
+  - name: GigabitEthernet0/0
+    type: 1000base-t
+    mgmt_only: true
+  - name: TenGigabitEthernet1/0/1
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/2
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/3
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/4
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/5
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/6
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/7
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/8
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/9
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/10
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/11
+    type: 10gbase-x-sfpp
+  - name: TenGigabitEthernet1/0/12
+    type: 10gbase-x-sfpp
+module-bays:
+  - name: Network Module
+    position: '1'
+console-ports:
+  - name: con0
+    type: rj-45
+power-ports:
+  - name: PSU0
```

Comment:
This appears to be a modular PSU. These should be put under module-bays and you should also create the fans as modules as well

It also appears there are 2 for this model